.github/workflows/test.yml

```diff
@@ -45,9 +47,24 @@ jobs:
         with:
           profile: minimal
           toolchain: ${{ matrix.rust }}
+          target: ${{ matrix.target }}
           override: true
+      - name: Install cargo cross
+        run: |
+          curl -L https://github.com/cross-rs/cross/releases/download/v0.2.4/cross-x86_64-unknown-linux-gnu.tar.gz | tar xvz
+          sudo mv cross cross-util /usr/bin
+          which cross
+      - name: Install compiler for armhf arch
+        if: matrix.target == 'armv7-unknown-linux-gnueabihf'
+        run: |
+          sudo apt-get update
+          sudo apt-get install gcc-arm-linux-gnueabihf
```

Comment:
isn't the point of `cross` to not have to do this?

Reply:
that's a good point, will try without it!